ERCS/erc-XXXX.md

```diff
@@ -0,0 +1,795 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 8107
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.